unity/Assets/Plugins/Countly/CountlyManager.cs

```diff
@@ -195,9 +196,15 @@ protected void EndSession(long duration)
 
       builder.Append("&session_duration=");
       AppendConnectionData(builder, duration.ToString());
+			Log ("Requesting session end");
+		try {
+				WebRequest www = WebRequest.Create(appHost + "/i?" +builder.ToString());
+				www.GetResponse().Close();
 
-      ConnectionQueue.Enqueue(builder.ToString());
```

Comment:
Let me confirm what you are saying:

- "queueUsesStorage" is enabled on CountlyManager 
- before the app quits, the app is paused and the "end session" request is put in the queue and a request is established
- the app quits before the request succeeds
- upon restart the queue is loaded, and the "end session" request is tried again 
- at the same time, the "session start" request is queued and sent

Is there anything wrong in this?